### ISSUE TITLE
matrix-bot-honoroit failed

### ISSUE THREAD

**Issue by ghost** [spantaleev/matrix-docker-ansible-deploy#1518]
```
TASK [matrix-bot-honoroit : Ensure honoroit image is built] ************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: docker.errors.APIError: 500 Server Error: Internal Server Error ("Cannot locate specified Dockerfile: docker/Dockerfile")
fatal: [matrix.domain.tld]: FAILED! => changed=false 
  msg: 'An unexpected docker error occurred: 500 Server Error: Internal Server Error ("Cannot locate specified Dockerfile: docker/Dockerfile")'
```
Latest commit on ARM64

**Comment by spantaleev** [comment]
Hopefully fixed by 817d6833bfc0. Please let me know if that's not enough!

**Comment by rdaries** [comment]
```  msg: 'Error building localhost/honoroit - code: 1, message: The command ''/bin/sh -c make build'' returned a non-zero code: 1, logs: [''Step 1/11 : FROM registry.gitlab.com/etke.cc/base AS builder'', '''', ''Pulling from etke.cc/base'', ''Digest: sha256:765ab5657cd2d2c98816afc1765777bb08762cc88898976b78ec8609dd188b64'', ''Status: Image is up to date for registry.gitlab.com/etke.cc/base:latest'', '' ---> b0dc7e5127f0'', ''Step 2/11 : WORKDIR /honoroit'', '''', '' ---> Using cache'', '' ---> f6770db95365'', ''Step 3/11 : COPY . .'', '''', '' ---> 808645eeb0ba'', ''Step 4/11 : RUN make build'', '''', " ---> [Warning] The requested image''s platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested", '' ---> Running in 5c16c0d0b3ce'', ''\x1b[91mstandard_init_linux.go:228: exec user process caused: exec format error'', ''\x1b[0m'', ''Removing intermediate container 5c16c0d0b3ce'']'```

my vars:
```
# Honoroit
matrix_bot_honoroit_enabled: true
#matrix_bot_honoroit_container_self_build: true
matrix_bot_honoroit_password: PASS
matrix_bot_honoroit_roomid: "!room:domain.tld"
```

**Comment by spantaleev** [comment]
It seems like building this image (as per its [Dockerfile](https://gitlab.com/etke.cc/honoroit/-/blob/6656c7a90661990581f85ae08e9604c7cb2cbf35/Dockerfile)) requires the `registry.gitlab.com/etke.cc/base` base image.

That image is likely available for `amd64` only. It should either be made into a multiarch image, or something else needs to be done (an `arm64` build of that image may be pushed as `registry.gitlab.com/etke.cc/base-arm64` or `registry.gitlab.com/etke.cc/base:latest-arm64`, etc.)

Seems like something for @rakshazi to work on. Until fixed, there's nothing that we can do to make self-building on non-`amd64` platforms work. Thank you for discovering things problem!

**Comment by rakshazi** [comment]
Thank you for mention, will try to add multiarch build and ping back here.

Unfortunately, I don't have an arm to test

**Comment by rakshazi** [comment]
base image has been updated with multiarch support.
Try now, please

PS: if it works as expected, do **not** close that ticket, I'm adding multiarch support to the honoroit image itself, so no need for selfbuild after that

**Comment by spantaleev** [comment]
> I'm adding multiarch support to the honoroit image itself, so no need for selfbuild after that

Having multiarch support is great!

It's also nice that it has self-building support anyway, for people who'd like to build some custom version from source.

**Comment by rdaries** [comment]
Honoroit bot installs without errors, but **!ho commands** do not work. Or I'm not doing things right.

<img width="855" alt="Screenshot 2022-01-09 at 11 07 46" src="https://user-images.githubusercontent.com/96979815/148676203-58dade85-1f12-49f9-bb98-4732047e363b.png">

**Comment by rakshazi** [comment]
Commands should be sent into the thread, not in room.

To enable threads support, you need toggle Threads in the Labs section of element client settings.

Honoroit doesn't work with standard replies (but feel free to add a feauture request in the bot repo issues)

**Comment by rdaries** [comment]
Thanks for the clarification @rakshazi, after a few tests everything looks fine and works perfectly. I just noticed that "m.reaction" not proxied. 
thanks again

**Comment by rakshazi** [comment]
Please, create an issue in honoroit repo regarding that.

Honoroit is not finished yet, so if you find any issue, missing feature or anything you want to add/change, you can create an issue in the honoroit repo to discuss it further.

**Comment by etkecc** [comment]
Fixed in #1522
